tiledb/bioimg/converters/base.py

```diff
@@ -246,7 +248,11 @@ def tile_to_tiledb(level_tile: Tuple[slice, ...]) -> None:
 
             # Write group metadata
             with tiledb.Group(output_path, "w") as group:
-                group.meta.update(reader.group_metadata, axes=input_axes.dims)
+                group.meta.update(
+                    reader.group_metadata,
+                    axes=input_axes.dims,
+                    version=".".join(map(str, bioimg_version[0:3])),
```

Comment:
Why not just `__version__`?

Reply:
Was just thinking/overthinking that we might want to eliminate `dirty` versions. Another thing is that even in that form patch version cannot dictate a specification change. I would assume spec would change only in minor-or even major version.